src/cli/src/cluster/releases.rs

```diff
@@ -1,6 +1,8 @@
 use crate::CliError;
 use super::*;
-use fluvio_cluster::list_releases;
+use fluvio_cluster::HelmClient;
+
+const DEFAULT_CHART_NAME: &str = "fluvio/fluvio-core";
```

Comment:
I believe we're using the name `fluvio/fluvio-app` now

Reply:
If I use `fluvio/fluvio-app`, I am not able to fetch any releases from helm. Is that expected? The command I use is `helm search repo --versions fluvio/fluvio-core`

Reply:
@sehz, For this, do we need to fetch both versions for `fluvio/fluvio-app` and `fluvio/fluvio-core` and merge the output? 
E.g do we need to combine all these versions and list them?
```
$ helm search repo --versions fluvio/fluvio-app --devel
NAME             	CHART VERSION	APP VERSION 	DESCRIPTION            
fluvio/fluvio-app	0.6.0-latest 	0.6.0-latest	A Helm chart for Fluvio

$ helm search repo --versions fluvio/fluvio-core
NAME              	CHART VERSION	APP VERSION	DESCRIPTION            
fluvio/fluvio-core	0.5.1        	           	A Helm chart for Fluvio
fluvio/fluvio-core	0.5.0        	           	A Helm chart for Fluvio
fluvio/fluvio-core	0.4.1        	           	A Helm chart for Fluvio
fluvio/fluvio-core	0.4.0        	           	A Helm chart for Fluvio
fluvio/fluvio-core	0.3.0        	0.3.0      	A Helm chart for Fluvio
fluvio/fluvio-core	0.2.3        	0.2.3      	A Helm chart for Fluvio```

Reply:
only need to support `fluvio-app` which is defined as constant at the top of crate.  `fluvio-core` is older version is deprecated.